types/serialization.go

```diff
@@ -268,6 +283,21 @@ func (s *State) FromProto(other *pb.State) error {
 	s.LastBlockID = *lastBlockID
 	s.LastBlockTime = other.LastBlockTime
 	s.DAHeight = other.DAHeight
+
+	s.NextValidators, err = types.ValidatorSetFromProto(other.NextValidators)
+	if err != nil {
+		return err
+	}
+	s.Validators, err = types.ValidatorSetFromProto(other.Validators)
+	if err != nil {
+		return err
+	}
+	s.LastValidators, err = types.ValidatorSetFromProto(other.LastValidators)
+	if err != nil {
+		return err
+	}
+	s.LastHeightConsensusParamsChanged = other.LastHeightConsensusParamsChanged
```

Comment:
this should be `LastHeightValidatorsChanged` and not `LastHeightConsensusParamsChanged`?

Reply:
yes it should be LastHeightValidatorsChanged.